dev/TeachingTip/TeachingTip_rs1_themeresources.xaml

```diff
@@ -4,22 +4,28 @@
+            <StaticResource x:Key="TeachingTipBorderBrush" ResourceKey="SystemControlTransientBorderBrush"/>
             <SolidColorBrush x:Key="TeachingTipTopHighlightBrush" Color="#0DFFFFFF"/>
-            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemChromeMediumHighColor"/>
-            <SolidColorBrush x:Key="TeachingTipStaticBackgroundBrush" Color="{ThemeResource SystemChromeLowColor}"/>
+            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
+            <StaticResource x:Key="TeachingTipForegroundBrush" ResourceKey="SystemControlForegroundBaseHighBrush"/>
+            <StaticResource x:Key="TeachingTipBackgroundBrush" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
+            <Thickness x:Key="TeachingTipAlternateCloseButtonBorderThickness">0</Thickness>
         </ResourceDictionary>
         <ResourceDictionary x:Key="Light">
-            <SolidColorBrush x:Key="TeachingTipBorderBrush" Color="#24000000"/>
+            <StaticResource x:Key="TeachingTipBorderBrush" ResourceKey="SystemControlTransientBorderBrush"/>
             <SolidColorBrush x:Key="TeachingTipTopHighlightBrush" Color="#99FFFFFF"/>
-            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemChromeMediumHighColor"/>
-            <SolidColorBrush x:Key="TeachingTipStaticBackgroundBrush" Color="{ThemeResource SystemChromeLowColor}"/>
+            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
+            <StaticResource x:Key="TeachingTipForegroundBrush" ResourceKey="SystemControlForegroundBaseHighBrush"/>
+            <StaticResource x:Key="TeachingTipBackgroundBrush" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
+            <Thickness x:Key="TeachingTipAlternateCloseButtonBorderThickness">0</Thickness>
         </ResourceDictionary>
         <ResourceDictionary x:Key="HighContrast">
-            <SolidColorBrush x:Key="TeachingTipBorderBrush" Color="Red"/>
-            <SolidColorBrush x:Key="TeachingTipTopHighlightBrush" Color="Black"/>
-            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemChromeMediumHighColor"/>
-            <SolidColorBrush x:Key="TeachingTipStaticBackgroundBrush" Color="{ThemeResource SystemChromeLowColor}"/>
+            <StaticResource x:Key="TeachingTipBorderBrush" ResourceKey="SystemControlTransientBorderBrush"/>
+            <SolidColorBrush x:Key="TeachingTipTopHighlightBrush" Color="Transparent"/>
+            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
+            <StaticResource x:Key="TeachingTipForegroundBrush" ResourceKey="SystemControlForegroundBaseHighBrush"/>
+            <StaticResource x:Key="TeachingTipBackgroundBrush" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
```

Comment:
We don't need to define TeachingTip_rs2_themeresources.XAML.

SystemControlTransientBackgroundBrush is already defined in Acrylic rs1 resource and TeachingTipBackgroundBrush can connect to it directly.
If it's just because of test case problem, UseNonstandardCondtionalXAML will make test ignore this file.

If you still have problem to handle TeachingTipBackgroundBrush, you may use:
You may use contract4NotPresent:StaticResource

Reply:
Agree. I tried doing that initially (you can see the earlier commit) but it broke ThemeResourcesTests. I'm going to go with this for now and then will do more refactoring across the projects as a follow-up.